packages/themes/tasks/builders/modules-content-switcher-tokens.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright IBM Corp. 2015, 2023
```

Comment:
https://github.com/carbon-design-system/carbon/issues/18954#issuecomment-2813348278

```suggestion
 * Copyright IBM Corp. 2025
```